vital/algo/metadata_filter.h

```diff
@@ -30,16 +32,17 @@ class VITAL_ALGO_EXPORT metadata_filter
   : public kwiver::vital::algorithm_def< metadata_filter >
 {
 public:
+  /// Algorithm can use the frame image for its operation.
+  ///
+  /// This capability indicates if the algorithm is able to make use of the
+  /// frame image. If this is not set, it implies that passing a non-null
```

Comment:
> If this is not set

Set to `false`, I think (since the default is `true`)? If so, might be good to clarify in the comment.

Reply:
I think this does mean "set to true" but the three negations is a bit confusing.

Reply:
TBH, I'm not following the comments here.

Is this better?
```
  /// This capability indicates if the algorithm is able to make use of the
  /// frame image. If this is not set, it implies that passing a null pointer
  /// as the input image to #filter will not affect the results, which may
  /// afford significant optimization opportunities to users.
```

Reply:
Yeah, this seems reasonable.

Reply:
Weak suggestion: "is not set" -> "is set to false". Approving changes in either case.

Reply:
Capabilities are actually tri-state: `true`, `false` and unset. The intent was that unset may be treated as `false`. (The intent is *also* "don't leave it unset", but...)

That being the case, maybe "is not \c true" is preferable?